example-config.yml

```diff
@@ -15,3 +15,4 @@ apps:
     hostname: www.local.bcc3d.ca
     google_analytics_key: UA-162929587-1
     google_maps_api_key:
+    jekyll_environment: #production
```

Comment:
A nicer pattern here is to use `production: true`, then use a ternary to insert the value `"production"`

The (very minor) problem with the current approach is that it involves a special value "production"

Reply:
Agreed, that's a much better pattern! I've made the necessary changes